recipes-kernel/linux/files/bcm2712-raspberrypi5-xen.dtso

```diff
@@ -60,3 +59,5 @@
 &bcm_reset      { xen,passthrough; };
 &pcie_rescal    { xen,passthrough; };
 &mip0           { xen,passthrough; };
+&rp1_usb0       { xen,passthrough; };
+&rp1_usb1       { xen,passthrough; };
```

Comment:
So usb are passthrough by default for both cases? But when mmc is configured - these devices are not present in domd DT

Reply:
fixed